imgpkg/002-recursive-bundles/examples/bundle-1/.imgpkg/images.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: imgpkg.carvel.dev/v1alpha1
+kind: ImagesLock
+images:
+- image: ghcr.io/k14s/design-docs/simple-app@sha256:4c8b96d4fffdfae29258d94a22ae4ad1fe36139d47288b8960d9958d1e63a9d0
+  annotations:
+    kbld.carvel.dev/id: my.registry.io/simple-application
+- image: ghcr.io/k14s/design-docs/utilities@sha256:47ae428a887c41ba0aedf87d560eb305a8aa522ffb80ac1c96a37b16df038e0f
```

Comment:
we would need to make sure that kbld is happy to skip images that arent annotated with `kbld.carvel.dev/id`.

Reply:
I just tried it and it does not look happy

```
$ cat tmp/images1.yml
---
apiVersion: imgpkg.carvel.dev/v1alpha1
images:
- annotations:
  image: gcr.io/cf-k8s-lifecycle-tooling-klt/demo-projects-app@sha256:27c54fe47e8b86576557100a4a0e0ee7c358629f40c6a7c93940f656aba7bcd2
kind: ImagesLock
```

```
$ kbld -f deployment/config/app.yml -f tmp/images1.yml
kbld: Error: Validating imageslock/ (imgpkg.carvel.dev/v1alpha1) cluster:
  Validating Overrides[0]:
    Expected Image or ImageRepo to be non-empty
```